lady_luck.js

```diff
@@ -129,16 +128,14 @@ function nwodRoll(pool, options='') {
 
 function suxxToWords(suxx) {
 	return suxx
-	  ? `${suxx} success${suxx == 1 ? '' : 'es'}`
-	  : 'No success'
+	  ? suxx + " success" + (succ == 1 ? '' : 'es')
```

Comment:
You renamed `suxx` to `succ`, but only in one specific location, which breaks the script due to a variable name not found error.

Reply:
lol